src/stream_base-inl.h

```diff
@@ -269,7 +269,21 @@ inline WriteWrap* StreamBase::CreateWriteWrap(
   return new SimpleWriteWrap<AsyncWrap>(this, object);
 }
 
-template <class Base>
+
+#define STREAM_BASE_METHOD(func, str)                                      \
+  do {                                                                     \
+    Local<FunctionTemplate> func_templ =                                   \
+      env->NewFunctionTemplate(func,                                       \
```

Comment:
Btw, we use 4 spaces of indentation here – and maybe it makes sense to turn this macro into a function?

Reply:
Tips: use `CLANG_FORMAT_START=master make format-cpp` to auto format, or replace `master` with some other commit that you are branching from